planning/autoware_surround_obstacle_checker/param/surround_obstacle_checker_node_parameters.yaml

```diff
@@ -0,0 +1,67 @@
+surround_obstacle_checker_node:
```

Comment:
@mitukou1109 Is it possible to create this yaml file under `config` dir...?

Reply:
Isn't it confusing if it is in the same directory as the original parameter file? The file is just for code generation.

Reply:
:thinking: OK. Let's use param dir for now.